ospfd/ospf_zebra.c

```diff
@@ -1353,11 +1353,18 @@ static int ospf_distribute_list_update_timer(struct thread *thread)
 						default_refresh = 1;
 					else if (
 						(lsa = ospf_external_info_find_lsa(
-							 ospf, &ei->p)))
```

Comment:
I think these changes belong to a different commit, would you mind to split it please?

Reply:
Sure, will split it.

Reply:
The split is done. 
Thanks,